validation-test/stdlib/Dictionary.swift

```diff
@@ -4751,6 +4763,22 @@ DictionaryTestSuite.test("Hashable") {
 }
 
 DictionaryTestSuite.setUp {
+#if _runtime(_ObjC)
+  // Exercise ARC's autoreleased return value optimization in Foundation.
+  //
+  // On some platforms, when a new process is started, the optimization is
+  // expected to fail the first time it is used in each linked
+  // dylib. StdlibUnittest takes care of warming up ARC for the stdlib
+  // (libswiftCore.dylib), but for this particular test we also need to do it
+  // for Foundation, or there will be spurious leaks reported for tests
+  // immediately following a crash test.
```

Comment:
😱

Reply:
Hm, maybe StdlibUnittest should do this instead, for every test. However, the relevant file doesn't import Foundation, and I hesitate to add the import there, for fear of even more unintended consequences.